### PR TITLE
Making Traits More Readable and Setting Better Setpoints for Water Heaters

### PR DESCRIPTION
**Pull request by barndawgie**
This traits code was really hard to follow; I think this is a bit better? I'm not sure why the earlier SETPOINT_MAX was set to temperature of the sun, but 110-140 Fahrenheit seems to be the actual limits of the Econet app.

### CODE REVIEW

**Comment by Daniel-dev22** [comment]
This may not be needed with the changes coming in #30. Users will also be able to configure entity category per entity in yaml.

**Comment by barndawgie** [comment]
Roger that. Let me see if this is handled in that revised version and rework my PR if needed.
